lib/main_common.pm

```diff
@@ -1944,7 +1947,8 @@ sub load_create_hdd_tests {
     # temporary adding test modules which applies hacks for missing parts in sle15
     loadtest 'console/sle15_workarounds' if is_sle('15+');
     loadtest 'console/integration_services' if is_hyperv;
-    loadtest 'console/hostname'              unless is_bridged_networking;
+    loadtest 'console/hostname' unless is_bridged_networking;
+    loadtest 'console/system_prepare';
```

Comment:
hmm, it seems #5555 should be revert and close. Because 'console/system_prepare' had done what I want to. I am running to test again.

Reply:
I wanted to check that first, to avoid potential duplication. Will follow-up on that. Thanks!